js/src/sdk/models/triggers.ts

```diff
@@ -29,16 +37,27 @@ export class Triggers {
    * @returns {CancelablePromise<ListTriggersResponse>} A promise that resolves to the list of all triggers.
    * @throws {ApiError} If the request fails.
    */
```

Comment:
The JSDoc comment for the `list` method needs to be updated to reflect the new parameter type `TTriggerQuery` and document the new filter options available.